test/circuit/input.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Seems this file is not used

Reply:
I would keep it for manual testing if someone would want to use it somewhere else.

Reply:
In that case I would use this file instead of https://github.com/iden3/circom_runtime/pull/112/files#diff-aa612a50d6149d64cfddc4428796606284524a819d56428906156fa342e5f26fR12-R16 object, but it's up to you